pysat/tests/test_files.py

```diff
@@ -1133,8 +1133,8 @@ def teardown(self):
                          temporary_file_list=self.temporary_file_list)
         pysat.params['data_dirs'] = self.data_paths
 
-# TODO: This needs to be replaced or expanded based on the tests that
-# portalocker uses
+# TODO(jklenzing): This needs to be replaced or expanded based on the tests that
+# portalocker uses -- #871
```

Comment:
The hacking site recommends including a name in parens.  We could just as easily use TODO(#871), which would satisfy the linter and our standards.